src/TMS/CourseCreation/LinkHelper.php

```diff
@@ -4,6 +4,8 @@
 
 namespace ILIAS\TMS\CourseCreation;
 
+require_once("./Services/User/classes/class.ilObjUser.php");
```

Comment:
Warum?

Reply:
Weil ich das sonst nicht Mocken kann. Und in Tests wollen wir require von ILIAS Komponenten ja  möglichst verzichten. Grund ist das Typehinting in den Funktion.